packages/web3/test/stress/large_request_data/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+. scripts/env.sh
```

Comment:
I think the `start.sh` for these should be in scripts folder, what do u think?

Reply:
as for me, it is better to keep all things together.